specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json

```diff
@@ -675,6 +675,49 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CognitiveServices/locations/{location}/checkDomainAvailability": {
+      "post": {
+        "tags": [
+          "CognitiveServicesAccounts"
+        ],
+        "operationId": "Locations_CheckDomainAvailabilityPost",
```

Comment:
it would be good if we can rename operation id to `Locations_CheckDomainAvailability`, no need to include underneath HTTP verb in the method name